fmriprep/workflows/bold/base.py

```diff
@@ -912,21 +935,21 @@ def init_func_derivatives_wf(output_dir, output_spaces, template, freesurfer,
                                            ('bold_mask_mni', 'in_file')]),
         ])
 
-    if freesurfer:
-        ds_bold_aseg_t1 = pe.Node(DerivativesDataSink(
-            base_directory=output_dir, space='T1w', desc='aseg', suffix='dseg'),
-            name='ds_bold_aseg_t1', run_without_submitting=True,
-            mem_gb=DEFAULT_MEMORY_MIN_GB)
-        ds_bold_aparc_t1 = pe.Node(DerivativesDataSink(
-            base_directory=output_dir,  space='T1w', desc='aparcaseg', suffix='dseg'),
-            name='ds_bold_aparc_t1', run_without_submitting=True,
-            mem_gb=DEFAULT_MEMORY_MIN_GB)
-        workflow.connect([
-            (inputnode, ds_bold_aseg_t1, [('source_file', 'source_file'),
-                                          ('bold_aseg_t1', 'in_file')]),
-            (inputnode, ds_bold_aparc_t1, [('source_file', 'source_file'),
-                                           ('bold_aparc_t1', 'in_file')]),
-        ])
+        if freesurfer:
+            ds_bold_aseg_mni = pe.Node(DerivativesDataSink(
+                base_directory=output_dir, space=template, desc='aseg', suffix='dseg'),
+                name='ds_bold_aseg_mni', run_without_submitting=True,
+                mem_gb=DEFAULT_MEMORY_MIN_GB)
+            ds_bold_aparc_mni = pe.Node(DerivativesDataSink(
+                base_directory=output_dir,  space=template, desc='aparcaseg', suffix='dseg'),
+                name='ds_bold_aparc_mni', run_without_submitting=True,
+                mem_gb=DEFAULT_MEMORY_MIN_GB)
+            workflow.connect([
+                (inputnode, ds_bold_aseg_mni, [('source_file', 'source_file'),
+                                              ('bold_aseg_mni', 'in_file')]),
```

Comment:
This open parenthesis should be aligned with the one in the line above, so just add one more space, like so:

```suggestion
                                               ('bold_aseg_mni', 'in_file')]),
```